---
layout: post
title: "New paper: Rapid host switching in Campylobacter"
author: "Bethany Dearlove"
tags: [research, Campylobacter, new paper]
image: campyplot.png
---

My new paper with [Daniel Wilson](www.danielwilson.me.uk/), [Sam Sheppard](https://sheppardlab.com/) and colleagues on [rapid host switching in generalist _Campylobacter_ strains](https://www.nature.com/articles/ismej2015149) has been published in The ISME Journal (open access). 

_Campylobacter_ is the most commonly identified cause of bacterial gastroenteritis worldwide - with more cases than _Salmonella_, _E. coli_, _Clostridium_ and _Listeria_ combined. Infection is generally associated with food poisoning, particularly the consumption of raw poultry products. However, the _Campylobacter_ genus comprises several widely distributed species of zoonotic pathogens, carried, often asymptomatically, in the gut microbiota of a range of mammalian and avian species, including, notably, those found on farms. As a consequence, opportunities for contamination of products including meat, poultry and unpasteurised milk can take place at any time from farmyard to fork. Although some strains of Campylobacter are only found in a single species, others appear associated with numerous host species. Indeed, some of the most common lineages associated with human disease are found frequently in both chicken and cattle, making it hard to infer the true source of infection. 

Previous work has focused on MLST typing, using only 7 genes representing about 1% of the full genome. The idea behind this new paper was that the increased resolution afforded by whole genome sequencing could reveal previously undetected differences between the strains found on chicken, versus those, say, on pork or beef. However, somewhat surprisingly, this was not found to be the case. Instead, we show that common strains of _C. jejuni_ and _C. coli_ infectious to humans seem to be rapidly switching between different hosts species - much more quickly than mutations can accumulate to differentiate sub-populations between those hosts. This suggests these lineages are adapted to a generalist lifestyle, permitting rapid transmission between different hosts.

This conclusion also highlights that whole genome sequencing by itself cannot substitute for intensive sampling of suspected transmission reservoirs when searching for the source of human infection.
